content/trial_getting_started/examples/kubernetes/_index.md

```diff
@@ -0,0 +1,99 @@
+
+### Prerequisites
+This example expects the following prerequisites:
+
+* A {{< param cfy_manager_name >}} setup ready. This can be either a [{{< param mgr_hosted_title >}}]({{< param mgr_hosted_link >}}), a [{{< param mgr_premium_title >}}]({{< param mgr_premium_link >}}), or a [{{< param mgr_community_title >}}]({{< param mgr_community_link >}}).
+* The following plugins must be uploaded to the {{< param cfy_manager_name >}}
+  * AWS Plugin (version 2.5.6+)
+  * Kubernetes Plugin (version 2.9.3+) 
+  * Terraform Plugin (version 0.15.0+)
+  * Fabric Plugin (version 2.0.7+)
+* The following secrets must be defined on the {{< param cfy_manager_name >}}
+  * `aws_access_key_id` - The AWS access key
+  * `aws_secret_access_key` - The AWS secret key
+  * `aws_keypair` - The name of an AWS keypair to associate with virtual machines.  The key must be created on AWS prior to running the example.
+  * `private_key_content` - The SSH private key contents from the keypair
+  
+* Access to the cloud infrastructure you select is required to demonstrate this example.  That can mean ability to allocate the required VMs and networking (ECS), and/or access to S3, RDS, and EKS.
+* These instructions use the CLI to run the example.  Using the CLI requires an addition [installation step](../../../install_maintain/installation/installing-cli.md) unless the example is run on the manager itself, un which case it is pre-installed.
+
+### Install the Example
+Our [Environment-as-a-Service example on GitHub](https://github.com/cloudify-community/eaas-example) demonstrates a deploy-time selectable Kubernetes-based environment that includes object and relational storage external to the Kubernetes cluster.  The three selectable environment types representing small and large development environments, and a production environment.
+
+* Download or clone the example to your local system.  If downloaded as an archive, the archive must be extracted.
+* Upload each blueprint in the `infra` directory, and use the names from the table below:
+
+| Path | Name |
+| ---- | ---- |
+| infra/dev/minikube | minikube |
+| infra/dev/minio | minio |
+| infra/dev/multi_node | multi_node |
+| infra/dev/psql | psql |
+| infra/dev/single_node | single_node |
+| infra/dev/vm | vm |
+| infra/prod/eks | eks |
+| infra/prod/prod_network | prod_network |
+| infra/prod/rds_psql | rds_psql |
+| infra/prod/s3 | s3 |
+| infra/vpc | vpc |
+
+* Upload the application/main blueprint from `app/blueprint.yaml`.
+* Create a small development cluster:
+  
+```
+cfy deployments create app_dev_small -b app -i env_type=dev-small
+cfy executions start install -d app_dev_small
+```
+
+* Create a large development cluster:
+
+```
+cfy deployments create app_dev_large -b app -i env_type=dev-large
+cfy executions start install -d app_dev_large
+```
+
+* Create a production cluster:
+
+```
+cfy deployments create app_prod -b app -i env_type=production
+cfy executions start install -d app_prod
+```
```

Comment:
I also tried to deploy this, and the existing instructions do not work. I think we are lacking instructions for deploying the infrastructure layer - which is good, because we need to change the example to use a different infrastructure later (see other comments.)

Reply:
Interesting.  I was just writing not testing or debugging.  I assumed it works.